docker_images/sklearn/app/pipelines/tabular_classification.py

```diff
@@ -1,18 +1,34 @@
+import json
+import warnings
+from pathlib import Path
 from typing import Dict, List, Union
 
 import joblib
 from app.pipelines import Pipeline
-from huggingface_hub import cached_download, hf_hub_url
+from huggingface_hub import snapshot_download
 
 
 DEFAULT_FILENAME = "sklearn_model.joblib"
 
 
 class TabularDataPipeline(Pipeline):
     def __init__(self, model_id: str):
-        self.model = joblib.load(
-            open(cached_download(hf_hub_url(model_id, DEFAULT_FILENAME)), "rb")
-        )
+        cached_folder = snapshot_download(repo_id=model_id)
```

Comment:
Why not use `skops` instead?

Reply:
I had two reasons in mind:

- skops would be an extra dependency, which we don't need if we're only downloading the model
- we don't care about caching here, if anything, it's a good thing and we don't care about where the model is downloaded.

So at the end it didn't seem like we really need the dependency. If in the future more features come in skops which we might need here, then we can change that.

Reply:
Oh I mean quite the other way around. In skops I wanted to make sure the cache is deleted and there is only one copy locally, and that it's on the path that the user decides. `snapshot_download` does caching and everything, so we're fine for here.

Reply:
It's already there ;)

Reply:
Great! Sorry, I just saw the diff. Thanks :+1: